File: List 3/1/aluno.ts
import { Pessoa } from "./pessoa";
import { Data } from "./data";

export class Aluno extends Pessoa{
     dataMatricula:Data

    constructor(nome: string,idade: number,dataNascimento: Data,dataMatricula: Data){
        super(nome,idade,dataNascimento);
        this.dataMatricula = dataMatricula
    }

}

File: List 3/2/aluno.py
from pessoa import Pessoa

class Aluno(Pessoa):
    def __init__(self,nome:str,sobrenome:str,matricula:int):
        super().__init__(nome,sobrenome)
        self.matricula = matricula

    def set_matricula(self,matricula):
        self.matricula = matricula

    def matricula(self):
        return self.matricula
